Cargo.toml

```diff
@@ -18,3 +18,6 @@ exclude = [
 [dependencies]
 xml-builder = "0.5.1"
 chrono = "0.4.22"
+
+[dev-dependencies]
+pretty_assertions = "1.3.0"
```

Comment:
Normally I try to limit the number of dependencies by only including the utmost essential ones. However, I recognize that these great integration tests you've written would be a huge pain in the ass to debug without this particular dependency, so I'm fine keeping it.